doc/themes/scikit-learn/layout.html

```diff
@@ -193,15 +193,14 @@
 	</div>
       </div>
     </div>
-    <div id="intro_to_sklearn_p" class="span6">
+    <div id="intro_to_sklearn_p" class="span5">
       <h1>scikit-learn</h1>
       <h2>Machine Learning in Python</h2>
-      <ul>
-        <li>Simple and efficient tools for data mining and data analysis</li>
-        <li>Accessible to everybody, and reusable in various contexts</li>
-        <li>Built on NumPy, SciPy, and matplotlib</li>
-        <li>Open source, commercially usable - BSD license</li>
-      </ul>
+      <p style="line-height: 1.5">
+        scikit-learn is a Python library for machine learning built on top of SciPy and
+        NumPy. scikit-learn contains models for supervised and unsupervised learning,
+        as well as tools for preprocessing, model selection and evaluation.
```

Comment:
I liked the bullet points... I find it easier to get the gist. Maybe we can say what you suggest with bullet points? Also I think that the license message is important and not obvious for new comers.

Reply:
I don't like the bullet points because they are hard to reuse as copy.

If people prefer them I'm fine to leave it as bullet points, but we should at least say what sklearn is.

What part of the license message, and what do you mean by new comers? For anyone from the scipy/pydata ecosystem I think BSD is assumed by default. Newcomers to python? To using open source tools? Who is the target audience?

Reply:
well obviously in this position it has a broad target audience.

I agree that I like the bullet points.

Reply:
Hm ok, my main point was to move away from bullets to a blurb but if neither of you likes that that's fine ;)
Maybe we can still have a blurb in the readme though?

What does broad target audience mean, though? Depending on what our definition of broad is, we should arguably explain what python is...

Reply:
+1 for a blurb in readme but keep the bullet points on front page. Sorry for the slow reaction time...